README.md

```diff
@@ -141,6 +141,44 @@ Create content artifacts.
 `demisto-sdk create -a .`
 This will create content artifacts in the current directory.
 
+## Upload
```

Comment:
Seems like a nice addition, and i think we should add an option to upload an integration/script package(the whole directory) so that the user won't have to do several steps in the way, or so that he will not get confused and will upload an empty yml file, what do you think?

Reply:
Still new to Demisto, should users go thru `unify` before uploading or is it possible to upload the whole directory?

Reply:
it is not possible to upload the whole directory, but i think that we can mask it out using the upload function, we can do the unify to the package ourselves using another argument such as `--unify` which will first unify the package and then upload it in the same command